src/gen_hook.erl

```diff
@@ -227,7 +227,7 @@ run_hook([Handler | Ls], Acc, Params, Key) ->
     hook_fn_ret() | safely:exception().
 apply_hook_function(#hook_handler{hook_fn = HookFn, extra = Extra},
                     Acc, Params) ->
-    safely:apply(HookFn, [Acc, Params, Extra]).
```

Comment:
It was the only place we've used `safely:apply` function btw :)

Reply:
I know, I could try to get rid of this module but there's an entire test-suite testing properties of this. So now this is scoped to test compilation only.